lib/scsi_defs.h

```diff
@@ -25,6 +25,13 @@ namespace scsi_defs {
 
 using LunAddress = uint64_t;
 
+enum class PageCode : uint8_t {
```

Comment:
There's a number of page code types in SCSI. It would be more organized/readable to call this LogPageCode